nixos/modules/services/games/terraria.nix

```diff
@@ -124,10 +134,12 @@ in
 
       serviceConfig = {
         User    = "terraria";
-        Type = "oneshot";
-        RemainAfterExit = true;
+        Type = "forking";
+        GuessMainPID = true;
+        CapabilityBoundingSet = "";
```

Comment:
The capability changes seem strictly unrelated.  I'd also expect this to be moot seing as the process runs as non-root to begin with, is there some way it can gain capabilities?